### PR TITLE
Add new CO for STEM mixing

### PR DESCRIPTION
**Pull request by acolombier**
This goes with [this PR](https://github.com/mixxxdj/mixxx/pull/13086). 

Happy to add the quick effect new COs on this PR too, otherwise will come in a follow up.

### CODE REVIEW

File: source/chapters/appendix/mixxx_controls.rst [hunk 1]
```diff
@@ -2482,6 +2482,24 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 1.10.0
 
 
+.. mixxx:control:: [ChannelN],stem_count
+
+   The number of stem track available for the loaded file. `0` means the deck is not a STEM deck.
+
+   :range: 0, 2..4, read-only
```
Comment (daschuer) [inline source/chapters/appendix/mixxx_controls.rst:2489]: Is 1 not in range? 
```suggestion
   :range: integer, 0...4, read-only
```

File: source/chapters/appendix/mixxx_controls.rst [hunk 2]
```diff
@@ -2482,6 +2482,24 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
...
+
+   .. versionadded:: 2.?.0
```
Comment (daschuer) [inline source/chapters/appendix/mixxx_controls.rst:2491]:
```suggestion
   .. versionadded:: 2.6.0
```

File: source/chapters/appendix/mixxx_controls.rst [hunk 3]
```diff
@@ -2482,6 +2482,24 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
...
+
+
+.. mixxx:control:: [ChannelNStemM],color
+
+   The color for the STEM `M` on the deck. The first stem color can be retrieved with `[ChannelNStem1],color`.
+
+   :range: 3-Byte :term:`RGB` color code (or -1), read-only
+
+   .. versionadded:: 2.?.0
```
Comment (daschuer) [inline source/chapters/appendix/mixxx_controls.rst:2500]:
```suggestion
   .. versionadded:: 2.6.0
```

File: source/chapters/appendix/mixxx_controls.rst [hunk 4]
```diff
@@ -4111,6 +4132,16 @@ The :mixxx:cogroupref:`[Skin]` group contains controls that are used to selectiv
...
 
 
+.. mixxx:control:: [Skin],show_stem_controls
+
+   Toggle the STEM control section.
+
+   :range: binary
+   :feedback: STEM control section is shown/hidden.
+
+   .. versionadded:: 2.?.0
```
Comment (daschuer) [inline source/chapters/appendix/mixxx_controls.rst:4142]:
```suggestion
   .. versionadded:: 2.6.0
```

**Review by daschuer — CHANGES_REQUESTED** [review]

File: source/chapters/appendix/mixxx_controls.rst (hunk 1 above)
Comment (acolombier) [reply source/chapters/appendix/mixxx_controls.rst:2489]: No, if a stem deck as only one stem, then this a traditional deck, with a single stereo pair :)

File: source/chapters/appendix/mixxx_controls.rst (hunk 2 above)
Comment (acolombier) [reply source/chapters/appendix/mixxx_controls.rst:2491]: :rocket:

**Pull request merged**